framework/queue/Job.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\queue;
+
+use yii\base\Object;
+
+/**
+ * Job is the base class for queue job classes.
+ *
+ * @author Alexander Kochetov <creocoder@gmail.com>
+ * @since 2.0.5
+ */
+abstract class Job extends Object
```

Comment:
@klimov-paul Let ask that question after see implementation. Btw do you saw any modern queue components implementation? Please check one.